opentelemetry-api/src/opentelemetry/context/__init__.py

```diff
@@ -138,20 +135,26 @@ def __repr__(self):
         asyncio.run(main())
 """
 
-import typing
-
 from .base_context import BaseRuntimeContext
 
 __all__ = ["Context"]
 
 
-Context = None  # type: typing.Optional[BaseRuntimeContext]
+def create_context() -> BaseRuntimeContext:
```

Comment:
Although this is useful for this module's testing, I don't think this is something that should be exposed - I don't think users are really supposed to be creating more than one context.